modules/doc/content/newsletter/2023/2023_04.md

```diff
@@ -25,6 +37,28 @@ default, any *other* application code used in a such a simulation
 should be either compatible with those operations, or should itself
 disable them.
 
```

Comment:
```suggestion

### General 2D-XY Mesh Line Cutting Capability

`XYMeshLineCutter` was added to enable general line cutting for a 2D mesh that only contains TRI3 and QUAD4 elements in the XY-plane. The new mesh generator trims off the fraction of the input mesh that is located on one side of a given straight line. The elements/nodes near the cutting line of the mesh are processed to ensure a smooth cross-section instead of a "zigzag" cross-section as generated by `PlaneDeletionGenerator`.

```

Reply:
@GiudGiud FYI

Reply:
Thanks @miaoyinb! I made a slight edit to link the documentation pages for both objects you mentioned, so that readers can get more context.